src/hooks/useClickOutside.ts

```diff
@@ -12,10 +12,11 @@ const useClickOutside = ({
   callback,
   handler = 'click',
   closeOnOutsideClick = true,
-}: Props): { isClickOutside: boolean } => {
+}: Props) => {
   const [isClickOutside, setClickOutside] = useState(false);
 
   useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Try to not use `any`

Reply:
yep, I cannot found the proper type for this event